### ISSUE TITLE
Call of duty black ops declassified stuck at loading screen

### ISSUE THREAD

**Issue by Naruhikaaltergame** [Vita3K/Vita3K-Android#112]
I have an issue when i played CODBOD sometimes it loads a game with horrible graphics and sometimes it stuck on loading screen or even dosen't want to run at all i was try on US and EU region of the game but result was same hopefully will fix in next update
Device i used:Poco x3 nfc
![Screenshot_2023-02-13-15-38-46-407_org vita3k emulator](https://user-images.githubusercontent.com/125255186/218411056-42fd99d9-8fed-4f13-84ec-0d3dca4b6b54.jpg)
![Screenshot_2023-02-13-15-38-01-383_org vita3k emulator](https://user-images.githubusercontent.com/125255186/218411203-908692d2-78cb-4fd3-99c2-0fbeb00fb1a8.jpg)
![Screenshot_2023-02-13-15-37-17-108_org vita3k emulator](https://user-images.githubusercontent.com/125255186/218411259-55ada222-0754-43de-8958-5056285f9625.jpg)

**Comment by AccordiaDusk** [comment]
> I have an issue when i played CODBOD sometimes it loads a game with horrible graphics and sometimes it stuck on loading screen or even dosen't want to run at all i was try on US and EU region of the game but result was same hopefully will fix in next update
> Device i used:Poco x3 nfc
> ![Screenshot_2023-02-13-15-38-46-407_org vita3k emulator](https://user-images.githubusercontent.com/125255186/218411056-42fd99d9-8fed-4f13-84ec-0d3dca4b6b54.jpg)
> ![Screenshot_2023-02-13-15-38-01-383_org vita3k emulator](https://user-images.githubusercontent.com/125255186/218411203-908692d2-78cb-4fd3-99c2-0fbeb00fb1a8.jpg)
> ![Screenshot_2023-02-13-15-37-17-108_org vita3k emulator](https://user-images.githubusercontent.com/125255186/218411259-55ada222-0754-43de-8958-5056285f9625.jpg)
> 
> 

Since your phone ain't a snapdragon one, you may expect this kind of issues while the emulator grows up. But anyways, this is just day one and already is capable of run some games even on low snapdragon chipsets, you can expect just the best in the near future for your device.

**Comment by Naruhikaaltergame** [comment]
Umm my smartphone it's Snapdragon 732g

**Comment by AccordiaDusk** [comment]
> Umm my smartphone it's Snapdragon 732g

Yeah, sorry. Wrong person. Already deleted the previous message. Have you tried with custom drivers?

**Comment by Naruhikaaltergame** [comment]
Already try it with turnip latest

**Comment by Naruhikaaltergame** [comment]
![Screenshot_2023-02-12-22-01-03-974_org vita3k emulator](https://user-images.githubusercontent.com/125255186/218449681-947d142f-6110-4333-8a57-e960eeec2777.jpg)

For graphic example it looks like this

**Comment by AccordiaDusk** [comment]
> Already try it with turnip latest

Which version are you testing? There are 3 that work different it seems. ![Screenshot_20230213-054729.png](https://user-images.githubusercontent.com/107894463/218450052-cf701436-d9fc-4292-89a6-ddc528dc0aff.png)

**Comment by Naruhikaaltergame** [comment]
Tested on PCSB00213 and PCSE00097

**Comment by Naruhikaaltergame** [comment]
both of them already tested it was same results on Android

**Comment by AccordiaDusk** [comment]
> Tested on PCSB00213 and PCSE00097 

That's weird. As far as I know, the dark textures were already fixed, at least in a previous desktop version according to this comment.
 https://github.com/Vita3K/compatibility/issues/465#issuecomment-1181822532

I guess we have to wait until it gets fixed on Android, or continue testing the version 213 with more drivers to see if there's one that works.

**Comment by Naruhikaaltergame** [comment]
I see

**Comment by greaserx** [comment]
i have the same graphical problem.

**Comment by Naruhikaaltergame** [comment]
Well in latest version still sometimes get running but black graphics but it's okay
![Screenshot_2023-02-15-08-55-08-166_org vita3k emulator](https://user-images.githubusercontent.com/125255186/218908131-e86c147e-1bac-4270-a217-636368dc1fb9.jpg)

**Comment by Phost12345** [comment]
Ddd

**Comment by sunilkumardasdev** [comment]
Having issues:
- Black screen flickering
- stuck on loading screen
- Black screen on loading 

On:
- PCSE00097
- SD870
- 6GB RAM
- A13
- Vita3K V0.2.0-12
